Makefile

```diff
@@ -20,17 +51,39 @@ tyrekicking-test: tmp/.tests-pass ## Test that the shell script can actually sta
 
 mutation-test: cypher-shell/build/reports/pitest/index.html ## Generate a mutation testing report
 
+clean: ## Clean build directories
+	rm -rf out
+	rm -rf tmp
+	$(GRADLE) clean
+
+rmhosts: ## Remove known hosts file
+	rm -rf ~/.neo4j/known_hosts
+
+launch: rmhosts clean build run ## Removes known hosts file, cleans, builds, and runs the shell
+
+prefix ?= /usr/local
+install: build cypher-shell.1 ## Install cypher-shell (requires pandoc for manual)
+	mkdir -p $(DESTDIR)/$(prefix)/bin
+	mkdir -p $(DESTDIR)/$(prefix)/share/cypher-shell/lib
+	mkdir -p $(DESTDIR)/$(prefix)/share/man/man1
```

Comment:
`$DESTDIR` comes from `rpmbuild` which defines it to be the temporary build directory.

https://www.gnu.org/software/make/manual/html_node/DESTDIR.html